src/lib/pages.ts

```diff
@@ -42,6 +42,7 @@ export default class Pages {
   savePage = (): void => {
     this.pagesJSON[this.currentIndex] = this.canvas.toObject([
       "id",
```

Comment:
This should probably be abolished in the future; store in the `data` property instead